packages/shared/src/components/widgets/SocialShareList.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { SocialShareIcon } from './SocialShareIcon';
+import CopyIcon from '../icons/Copy';
+import {
+  getEmailShareLink,
+  getFacebookShareLink,
+  getLinkedInShareLink,
+  getRedditShareLink,
+  getTelegramShareLink,
+  getTwitterShareLink,
+  getWhatsappShareLink,
+  ShareProvider,
+} from '../../lib/share';
+import TwitterIcon from '../icons/Twitter';
+import WhatsappIcon from '../icons/Whatsapp';
+import FacebookIcon from '../icons/Facebook';
+import RedditIcon from '../icons/Reddit';
+import LinkedInIcon from '../icons/LinkedIn';
+import TelegramIcon from '../icons/Telegram';
+import MailIcon from '../icons/Mail';
+import MenuIcon from '../icons/Menu';
+import { IconSize } from '../Icon';
+
+interface SocialShareListProps {
+  link: string;
+  description: string;
+  isCopying?: boolean;
+  onCopy?(): void;
+  onNativeShare(): void;
+  onClickSocial(provider: ShareProvider): void;
+}
+
+export function SocialShareList({
+  link,
+  description,
+  isCopying,
+  onCopy,
+  onNativeShare,
+  onClickSocial,
+}: SocialShareListProps): ReactElement {
+  return (
+    <>
+      {onCopy && (
+        <SocialShareIcon
+          onClick={onCopy}
+          icon={
+            <CopyIcon
+              className={classNames('text-theme-label-invert')}
+              secondary={isCopying}
+            />
+          }
+          className="btn-primary"
+          label={isCopying ? 'Copied!' : 'Copy link'}
+        />
+      )}
+      <SocialShareIcon
```

Comment:
The sizes match of the icons between the two implementations?
Didn't check, but the invite page one seem bigger then I recall (48x48px)

Reply:
Yes, they are 48x48 in both cases.